### DISCUSSION TITLE
Let's drop the policy requiring permission to change author metadata

### DISCUSSION THREAD

**Discussion by mjpost** [acl-org/acl-anthology#1873] [General]
A while back, we instituted [a policy](https://aclanthology.org/info/corrections/#note-on-author-changes) requiring authors to get permission from workshop or conference chairs in order to change any author information on papers. This was a blanket policy adopted in response to an ACL policy not allowing author changes once a paper was submitted. Our reasoning was that, were we not to require this permission, there was a loophole around this ACL policy: authors could submit the PDF as they wanted, and then later request that the metadata be updated to the PDF.

I propose we drop this policy, under the following reasoning:

* This policy is not uniform across workshops and conferences. 
* It is furthermore beyond our ability to track which conferences have this policy.
* This conference organizer usually has no idea about the policy, is not motivated to care, and gives permission perfunctorily.
* Enforcing it is time-consuming.
* If ACL (or any other conference) wants this policy, it should educate its publication chairs to close the loophole by ensuring that PDF author order matches the submitted metadata.

I therefore plan to drop this policy at the end of April, unless someone convinces me otherwise.

**Comment by mjpost** [comment]
Not sure who gets notified of this, so tagging @davidweichiang @danielgildea @mbollmann @akoehn.

**Reply by akoehn** [reply]
Everyone who has subscribed to new issues gets notified, which at least includes me :-)

**Reply by mbollmann** [reply]
I didn't get notified here before getting tagged, and I get notified for all new issues :thinking:

**Comment by davidweichiang** [comment]
Can you clarify what the new policy would be?

**Reply by mjpost** [reply]
I made this explicit in #1874.

**Comment by mjpost** [comment]
We take the PDF as authoritative, and process metadata corrections without question.

I think we do not allow authors to be added or removed, however, without external corroboration or permission.

**Comment by akoehn** [comment]
I am fine with allowing metadata changes to match the PDF, but not other changes.

Additionally, we should publicize the change and make organizers aware that it is their responsibility that the PDFs are correct.

**Comment by nschneid** [comment]
Publication chairs ensuring PDF matches metadata - Any thoughts on how this can be implemented at scale for large venues?

For SemEval we sought to achieve this by (a) adding several checkboxes to the camera-ready submission form to ask authors to check that author info matches the PDF, and (b) deputizing 2-3 student volunteers as publication assistants to go through the proceedings and look for issues. Not sure what would be appropriate for a conference with an order of magnitude more papers.

**Reply by nschneid** [reply]
FTR, I dug up the list of questions:

> 1. I/we have carefully considered the reviews and improved the paper as much as possible, exercising authorial discretion.
> 2. If the research involves data or systems for particular languages, those languages are named. (#BenderRule)
> 3. If the research raises ethical considerations (e.g. potential for misuse), these are discussed in the paper.
> 4. I have checked that the order/spelling/capitalization of author names above is correct and matches the PDF. (Authors registered in START may need to edit their names under account profile settings.)
> 5. The mandatory license footnote described in the style guide is included.

**Reply by mjpost** [reply]
I agree [with @akoehn](https://github.com/acl-org/acl-anthology/discussions/1873#discussioncomment-2578909) that this is not our problem. Conferences themselves are perfectly capable of closing this loophole, and it makes more sense that they bear the costs incurred by their own policies. Most of the corrections we process are harmless, and us enforcing the policy is adding a lot of friction to this process.

**Reply by mbollmann** [reply]
Is sanity-checking the final version PDF not part of the publication pipeline anyway? I've received e-mails from pub chairs telling me that my final version violated some page margins, or had a page break they wanted removed. Surely the author names could be checked as part of that process?

**Reply by mjpost** [reply]
I doubt it. It's a time-consuming process, with the things to check only growing. For example, when I was pub chair in 2013 for NAACL, we added a manual check that figures were B&W-readable, and went through for margin violations. [aclpubcheck](https://github.com/acl-org/aclpubcheck) is making some aspects of this easier, but there are also a lot of papers these days, especially for main conference proceedings. These policies tend to be created without much thought for enforcement or the costs imposed, IMO.

**Comment by akoehn** [comment]
I would just say that this is not our problem. The conferences only need to check this if they want to have control over the author list.

As we had zero cases where the change request was rejected, I don't see why we should have all the bureaucracy on our side.
